packages/site/src/lib/glosses/glossing-languages.ts

```diff
@@ -943,6 +943,24 @@ export const glossingLanguages: IGlossLanguages = {
   zu: {
     vernacularName: 'isiZulu',
   },
+  tri: {
```

Comment:
I added manually the new glosses, but I'm not sure if that's the right way.

Reply:
The should be added in the i18n spreadsheet. That will be transferred into the JSON from which you copy into here. But now Vite supports JSON natively without needing a plugin so I will adjust things here to just use the JSON directly. So then we will just need to update the i18n spreadsheet as our source of truth.